db/pictures.csv

```diff
@@ -21,4 +21,4 @@ leningradensis.jpg,1010,6.6,15,17.9,300,358,1008,
 amarna.jpg,-1350,23.5,14,15.44,300,331,Amarna EA 330,
 iphone.jpg,2007,45.7,7,13.158,532,1000,2007,
 hammurabi.jpg,-1870,10.8,11,12.17,300,332,0,
-lament_for_ur.jpg,-2020,7,11,17.16,250,390,lament for Ur,
+lament_for_ur.jpg,-2020,8,11,17.16,250,390,lament for Ur,
```

Comment:
further down to not overlap in Vietnamese, and plenty of space in other languages, just keep it at this locationi